client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -99,7 +114,7 @@ export class CommentNavigation extends Component {
 				<CommentNavigationTab className="comment-navigation__bulk-count">
 					<FormCheckbox
 						checked={ isSelectedAll }
-						onChange={ toggleSelectAll }
+						onChange={ this.toggleSelectAll }
```

Comment:
Unfortunately yes.
In `CommentNavigation` we now have the `getSiteComment` selector to build the minimum comment object (id, post id, status, i_like), then used by the change status method in `CommentList`.

The minimum comment object is needed both to have all the needed info to actually dispatch the action (comment id and post id) but also to be able to revert the action (previous status, previous i_like).

Undo is not the case for now with bulk actions, but I'd rather have the object ready and consistent with the whole "single action" flow.